src/store/modules/search/types.js

```diff
@@ -0,0 +1,11 @@
+const REQUEST_SEARCH = 'REQUEST_SEARCH';
+const SEARCH_ARTICLES_SUCCESS = 'SEARCH_ARTICLES_SUCCESS';
```

Comment:
Also, this isn't properly namespaced
```suggestion
const SEARCH_ARTICLES_SUCCESS = 'vali-ah-frontend/search/SEARCH_ARTICLES_SUCCESS';
```
Same feedback for the other action types